absearch/server.py

```diff
@@ -209,6 +209,9 @@ def handle_500_error(code):
         logger.error('An error occured. Sentry id: %s' % ident)
     logger.exception('An error occured')
 
+@app.error(404)
+def handle_404_error(code):
+    response.content_type = 'application/json'
```

Comment:
This does not look like the original issue. Is this really responsible of returning some html?

Reply:
I look up the docs and found https://bottlepy.org/docs/dev/tutorial.html#error-pages
My understanding is that Bottle has its own default 404 response and we could override it? Originally I had it returned nothing, but it felt weird that it was returning nothing with response content type text/html so I changed the content type to application/json?